src/components/Layout/Layout.js

```diff
@@ -49,7 +50,24 @@ const Layout = ({ children }) => {
         <PrimaryNav />
       </div>
       <main className="l-main" id="main-content">
-        {children}
+        <div data-test="main-children">{children}</div>
+        {!localStorage.getItem("release20_04") && (
+          <Notification
+            type="information"
+            close={() => {
+              localStorage.setItem("release20_04", true);
+            }}
+          >
+            Welcome the new JAAS Dashboard! This dashboard replaces the previous
```

Comment:
This wording isn't appropriate for all the environments we deploy this to.

Reply:
This is useless feedback, please be more specific - what's not appropriate about it?

Reply:
I had commented in my suggestion previously, but the issue is that this dashboard build is deployed in different environments so the messaging needs to be appropriate for those environments. So maybe something like...

```html
Welcome the new JAAS Dashboard! This dashboard is the replacement for the Juju GUI in JAAS and individual Juju Controllers from Juju 2.8.{" "}
<span className="u-hide--small">
  Read more and join the discussion about this new product{" "}
  <a href="#_">on Discourse</a>. We would love to hear your feedback!
</span>
```